apps/core/models/snippets.py

```diff
@@ -0,0 +1,35 @@
+from django.db import models
+from django.utils.translation import gettext as _
+from modelcluster.fields import ParentalKey
+from modelcluster.models import ClusterableModel
+from wagtail.admin.panels import FieldPanel, InlinePanel
+from wagtail.fields import RichTextField
+from wagtail.models import Orderable
+from wagtail.snippets.models import register_snippet
+
+
+@register_snippet
+class FooterContent(ClusterableModel):
```

Comment:
Oh and before I forget: please also make this class and `FooterItem` extend `wagtail.models.TranslatableMixin`, and regenerate the migrations so we can make sure that they are translatable